pydantic/main.py

```diff
@@ -301,13 +307,8 @@ class BaseModel(metaclass=ModelMetaclass):
         __signature__: 'Signature'
 
     Config = BaseConfig
-    __slots__ = ('__dict__', '__fields_set__')
-    # equivalent of inheriting from Representation
-    __repr_name__ = Representation.__repr_name__
-    __repr_str__ = Representation.__repr_str__
-    __pretty__ = Representation.__pretty__
-    __str__ = Representation.__str__
-    __repr__ = Representation.__repr__
+    __slots__ = ('__fields_set__',)
```

Comment:
please don't change this, if you think it needs changing, please create another issue.

Reply:
It was complaining because `Representation` already had `'__dict__'` in its `__slots__` (not allowed to have the same slot twice in the inheritance hierarchy), but now that you point this out I think the better solution is to set `__slots__ = tuple()` in `Representation`, then we can keep `'__dict__'` here 🙂